packages/subgraph/tests/utils/events/plugin.ts

```diff
@@ -1,22 +1,332 @@
-import {NumberStored} from '../../../generated/templates/Plugin/Plugin';
-import {Address, BigInt, ethereum} from '@graphprotocol/graph-ts';
-import {newMockEvent} from 'matchstick-as';
+import {DaoPlugin, MultisigProposal} from '../../../generated/schema';
+import {
+  ProposalCreated,
+  Approved,
+  ProposalExecuted,
+  MembersAdded,
+  MembersRemoved,
+  MultisigSettingsUpdated,
+} from '../../../generated/templates/Plugin/Plugin';
+import {ADDRESS_ONE, DAO_ADDRESS, CONTRACT_ADDRESS} from '../constants';
+import {
+  generatePluginEntityId,
+  generateProposalEntityId,
+} from '@aragon/osx-commons-subgraph';
+import {Address, BigInt, Bytes, ethereum} from '@graphprotocol/graph-ts';
+import {createMockedFunction, newMockEvent} from 'matchstick-as';
 
-export function createNewNumberStoredEvent(
-  number: string,
+// test data
+
+export const ZERO = '0';
+export const ONE = '1';
+export const TWO = '2';
+export const THREE = '3';
+
+export const PLUGIN_PROPOSAL_ID = ZERO;
+export const MIN_PROPOSER_VOTING_POWER = ZERO;
+export const START_DATE = '1644851000';
+export const END_DATE = '1644852000';
+export const CREATED_AT = ONE;
+export const SNAPSHOT_BLOCK = '100';
+export const ALLOW_FAILURE_MAP = '1';
```

Comment:
perhaps we could move these to a constant file for all the tests, WDYT?

Reply:
Is `ZERO` really better than writing '0'? I would rather think about eliminating most of these constants and refactor subgraph tests to have this state more locally defined.
I wouldn't avoid creating global constants like this that are just used inside tests.

Reply:
I suggest that the tests are refactored to be more unit test like.
Right now, the tests seem to be test everything the handler is doing.
Maybe, these tests can be broken down into smaller units.
Should we create a task @Rekard0 ?

Reply:
Moved to a separate task